pkgs/applications/window-managers/hyprwm/hyprland-plugins/default.nix

```diff
@@ -26,6 +26,7 @@ let
 
   plugins = lib.mergeAttrsList [
     { hy3 = import ./hy3.nix; }
+    { hyprsplit = import ./hyprsplit.nix; }
```

Comment:
Should be ordered alphabetically, so after hyprspace.

Reply:
corrected